client/landing/stepper/declarative-flow/internals/analytics/record-design.ts

```diff
@@ -90,3 +91,29 @@ export function getVirtualDesignProps( design: Design ) {
 		slug: design.is_virtual ? design.recipe?.slug : design.slug,
 	};
 }
+
+/**
+ * Tracks prop
+ *  name: assembler_source
+ *  values:
+ *		• virtual-theme
+ *		• blank-canvas-theme
+ * 		• standard
+ * 		• premium
+ * 		• default
+ */
+export function getAssemblerSource( design?: Design ) {
+	const { design_type, is_virtual } = design ?? {};
+
+	if ( is_virtual ) {
+		return 'virtual-theme';
+	}
+
+	if ( design_type === 'assembler' ) {
+		// blank-canvas-theme
+		return 'design-your-own';
+	}
+
+	// Standard, premium, default...
+	return design?.design_type;
```

Comment:
@autumnfjeld I decided to return `design.design_type` which has the theme type: Standard, premium, default...